src/app/api/test-types/model/testTypeCategory.ts

```diff
@@ -14,7 +14,7 @@ import { TestType } from './testType';
 /**
  * A category that can be composed of zero/more categories + one/more test types
  */
-export interface TestTypeCategory { 
```

Comment:
would be nice to update the swagger doc to reflect these changes (if it hasn't been done)

Reply:
Done: [#165](https://github.com/dvsa/cvs-svc-test-types/pull/165)